core/frontend/src/DisplayStyleState.ts

```diff
@@ -287,6 +287,15 @@ export abstract class DisplayStyleState extends ElementState implements DisplayS
     return undefined !== model && this.settings.contextRealityModels.delete(model);
   }
 
+  /** Detach the first [ContextRealityModel]($common) that matches the specified name and url.
+   * @see [DisplayStyleSettings.contextRealityModels]($common)
+   * @see [ContextRealityModels.delete]($common)
+   */
+  public detachRealityModelByKey(rdSourceKey: RealityDataSourceKey): boolean {
```

Comment:
These new methods look like they belong on DisplayStyleSettings, not DisplayStyleState.
(I know, there's a whole slew of reality-data-related methods on DisplayStyleState, but they're mostly not great APIs and don't belong there either).

Reply:
I marked them as internal for now so I can make DTA to work again. Not sure I want to keep them anyway.

Reply:
Adding "temporary" things to DisplayStyleState just to get something working is exactly how the display style reality model APIs became such a mess in the first place. If you need these new methods, please put them in the correct place.

Reply:
Moved in Display-test-app